Updated Tables Page

ReedGraff opened the pull request:
I changed the table page to better match the descriptions which were laid out.

V1EngineeringInc commented:
I like it. I have a couple small changes I want to make but I have been a bit wrapped up in a backend project. I will need a couple of days to get to this.

I really do appreciate it, thank you!

jamincollins commented:
I would love to see this published, I've been looking for ideas like this.

jamincollins commented:
Are there any additional details that can be provided around the table shown in the `docs/img/MetalTubing.gif`?

V1EngineeringInc commented:
I have actually added a bit of info since this has been submitted.

I don't know anything about metaltubing.gif and is exactly why I have been hesitant to merge something like this. I am willing to put a link but if it is in the docs like this I feel the need to be fully supportive of it.

ReedGraff commented:
> Are there any additional details that can be provided around the table shown in the `docs/img/MetalTubing.gif`?

Yes, there is, the table in the gif is a table I designed and built. The reason for it being able to swivel is that a full table wouldn't fit in my door unless I flipped it.

ReedGraff commented:
> I have actually added a bit of info since this has been submitted.
> 
> I don't know anything about metaltubing.gif and is exactly why I have been hesitant to merge something like this. I am willing to put a link but if it is in the docs like this I feel the need to be fully supportive of it.

No worries, I removed the gif and replaced it with an image of one of the more supported designs. If you or @jamincollins were interested in learning more, however, feel free to email me about it here: reed@langmuirsystems.com
